server/area.py

```diff
@@ -1720,9 +1720,21 @@ def change_status(self, value):
             "recess",
             "gaming",
         )
-        if value.lower() not in allowed_values:
+        value = censor(
+            value,
+            self.server.censors["whole"],
+            self.server.censors["replace"],
+            True,
+        )
+        value = censor(
+            value,
+            self.server.censors["partial"],
+            self.server.censors["replace"],
+            False,
+        )
+        if value.lower() == "hub":
             raise AreaError(
-                f'Invalid status. Possible values: {", ".join(allowed_values)}'
+                f'Invalid status. Possible values: {", ".join(possible_values)}'
```

Comment:
Maybe just outward specify "hub" is a restricted value?

Reply:
So just specify in ooc "Hub" is a restricted status?